src/components/@core/navigation-menu/light/left-menu/logo.tsx

```diff
@@ -23,15 +26,21 @@ const Logo = styled.div`
     }
   }
 
-  button {
+  button.ibpmenu {
     display: none;
-    padding: 0.5rem;
+    padding: 0.5rem 1rem;
```

Comment:
This button is for hamburger menu on mobile modifying it will make this button inconsistant

![image](https://user-images.githubusercontent.com/5774849/86358710-938b1e00-bc8d-11ea-80a2-fb2a4102e568.png)